operator/pkg/helm/vfs_renderer.go

```diff
@@ -30,9 +30,17 @@ import (
 const (
 	// DefaultProfileFilename is the name of the default profile yaml file.
 	DefaultProfileFilename = "default.yaml"
-
-	ChartsSubdirName = "charts"
-	profilesRoot     = "profiles"
+	ChartsSubdirName       = "charts"
+	profilesRoot           = "profiles"
+
+	// LocalBuildInfo provides instruction on how to build istio
+	// if the compiled in charts are not found
+	LocalBuildInfo = `
+ compiled in charts not found in this development build,
+ use manifests as local charts instead e.g:
+ - istioctl install --set hub=gcr.io/istio-testing -d manifests/
+ - or istioctl operator init --hub=gcr.io/istio-testing -d manifests/
```

Comment:
should this not be `--set hub=...`?

Reply:
`--set` works with `install` but not with `operator init`. I think there is an inconsistency between how `hub` is used across other commands. Anyways I have tested both the commands.